app/src/main/java/umc/everyones/lck/presentation/login/SignupMyteamFragment.kt

```diff
@@ -1,62 +1,67 @@
 package umc.everyones.lck.presentation.login
 
+import android.net.Uri
+import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.findNavController
-import androidx.navigation.fragment.navArgs
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
 import umc.everyones.lck.R
 import umc.everyones.lck.databinding.DialogMyteamConfirmBinding
 import umc.everyones.lck.databinding.FragmentSignupMyteamBinding
 import umc.everyones.lck.presentation.base.BaseFragment
-
-import android.widget.ImageView
-import androidx.core.content.ContextCompat
 import umc.everyones.lck.util.TeamData
 
 @AndroidEntryPoint
 class SignupMyteamFragment : BaseFragment<FragmentSignupMyteamBinding>(R.layout.fragment_signup_myteam) {
 
     private var selectedTeamName: String? = null
-    private val args: SignupMyteamFragmentArgs by navArgs()
+    private val viewModel: SignupViewModel by activityViewModels()
+    private val navigator by lazy { findNavController() }
 
     override fun initObserver() {
-        // No observers needed here
+        // No specific observers are needed for now
     }
 
     override fun initView() {
-        val nickname = args.nickname
-        val profileImageUri = args.profileImageUri
+        val profileImageUri = viewModel.profileImageUri.value?.toString()
 
-        setupTeamSelection { teamName ->
-            selectedTeamName = teamName
-        }
+        setupTeamSelection()
 
         binding.ivSignupMyteamNext.setOnClickListener {
             if (selectedTeamName == null) {
-                showTeamConfirmDialog(nickname, profileImageUri)
+                showTeamConfirmDialog(profileImageUri)
             } else {
-                navigateToSignupSuccess(nickname, profileImageUri, selectedTeamName)
+                lifecycleScope.launch {
+                    // User 추가
+                    viewModel.addUser(profileImageUri ?: "", selectedTeamName ?: "default_team")
+
+                    findNavController().navigate(R.id.action_signupMyteamFragment_to_signupSuccessFragment)
```

Comment:
```suggestion
                    navigator.navigate(R.id.action_signupMyteamFragment_to_signupSuccessFragment)
```
전역변수로 선언한 navigator를 사용해서 navigate 해주세여!!